xUnitTests/AddPositiveTest.cs

```diff
@@ -0,0 +1,19 @@
+using CSharpCalculator;
+
+namespace xUnitTests;
+
+public class AddPositiveTest
+{
+    [Fact]
+    public void Add10Plus20Expect30()
```

Comment:
Rename the tests. Do not add certain parameters to the name, as you may have different test data.